test/tests.yml

```diff
@@ -142,6 +187,17 @@
     - name: azure_rockylinux_stable
     - name: gce_default_stable
 
+- scenario: upgrade_containerd
+  initVersion: v1.28.7
+  upgradedVersion: v1.29.2
+  infrastructures:
+    - name: azure_centos_stable
+    - name: azure_default_stable
+    - name: azure_flatcar_stable
+    - name: azure_rhel_stable
+    - name: azure_rockylinux_stable
+    - name: gce_default_stable
```

Comment:
The `upgrade_containerd` scenario is based on the in-tree CCM and Kubernetes doesn't support in-tree CCMs as of 1.29. That said, this scenario/version combination is not possible at all (at least not in KubeOne).

Reply:
Just to confirm if I understand it correctly. Kubernetes has dropped in-tree ccms in favor of having them as a separate projects (e.g. kubernetes/cloud-provider-aws). 
And for us that means we don't test these for Kubernetes >= 1.29. Specifically the old scenarios are superseeded by the `upgrade_containerd_external` & `install_containerd_external` which use the corresponding external ccm depending on what the infrastructure keyword is that is provided in the test. In addition the csi_ccm_migration is completely.

Reply:
Yes, that's correct. All tests should be using `_external`-based scenarios if they're testing Kubernetes >= 1.29.